internal/core/src/index/ScalarIndex-inl.h

```diff
@@ -67,7 +67,8 @@ inline void
 ScalarIndex<std::string>::BuildWithRawData(size_t n, const void* values, const Config& config) {
     // TODO :: use arrow
     proto::schema::StringArray arr;
-    arr.ParseFromArray(values, n);
+    auto ok = arr.ParseFromArray(values, n);
+    Assert(ok);
```

Comment:
Maybe use `AssertInfo` to print some message here?

Reply:
Just check the proto and there is no way to get detailed error info.
We can only change proto log if we had to